### DISCUSSION TITLE
Quero somar 2 valores em que serão colocados em um input. (PYTHON)

### DISCUSSION THREAD

**Discussion by peepkyoto** [backend-br/forum#140] [Geral]
Quero somar 2 valores em que serão colocados em um input.
Alguém pode me explicar como faço isso?
CODIGO EM PYTHON:

print ("Seja bem vindo a loja do Pedro, veja os kits e coloque em seu carrinho");
print ("Temos alguns kits: Kit de foguete, Kit de net e kit de madeira")
foguete = 1
net = 3
madeira = 10
Kit = str(input("Qual seu primeiro kit? "))
Kit2 = str(input("Qual o segundo kit? "))

Total = (Kit + Kit2)
print (Total)

**Comment by eltinhosp-eng** [comment]
Kit = str(input("Qual seu primeiro kit? "))
Kit2 = str(input("Qual o segundo kit? "))

Bom dia Pedro, vc está passando uma string(str), vc tem que passar um
float, pois daí ele consegue somar as variáveis!

Em qua., 5 de abr. de 2023 18:32, eupedrolemos ***@***.***>
escreveu:
